common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.alibaba.fescar.common.thread;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+/**
+ * @author Otis.z
+ * @date 2019/2/26
+ */
+public class NamedThreadFactoryTest {
+
+    @Test
+    public void testNewThread() {
+        NamedThreadFactory namedThreadFactory = new NamedThreadFactory("testNameThread", 5);
+
+        Thread testNameThread = namedThreadFactory
+            .newThread(() -> {
+                try {
+                    Thread.sleep(1000);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
```

Comment:
Don't use `e.printStackTrace();` here.
We should declare the `throws` for this method:
```Java
public void testNewThread() throw InterruptedException {
```

Reply:
thx.i will be fixed on next commit